src/cloud-search-index-storage.js

```diff
@@ -47,15 +47,15 @@ export class CloudSearchIndexStorage {
   ]);
   #INDEX_PREFIX = 'company';
   #log;
-  #index;
+  index;
   #indexName;
 
   constructor(context, companyId) {
     this.#indexName = context.env.ALGOLIA_CI_INDEX || `${this.#INDEX_PREFIX}-${companyId}`;
     this.#log = new ContextHelper(context).getLog();
     this.#log.info('Using Search Index', this.#indexName);
 
-    this.#index = this.getClient(context).initIndex(this.#indexName);
+    this.index = this.getClient(context).initIndex(this.#indexName);
```

Comment:
this is required for the scripts so we don't need to make changes to this library everytime we want to run a command that is not already supported. lmk what you think